web/main/templates/export/as_printable_html/node.html

```diff
@@ -33,18 +33,22 @@ <h2 class="subtitle">{{ node.subtitle }}</h2>
 
 
     {% if node.is_resource %}
-      <section class="resource">
+     {# Whitespace beteen the <section> element and the content must not be introduced to avoid throwing off the annotation offset values #}
```

Comment:
```suggestion
     {# Whitespace between the <section> element and the content must not be introduced to avoid throwing off the annotation offset values #}
```